includes/sanitizers/class-amp-meta-sanitizer.php

```diff
@@ -92,38 +97,35 @@ static function ( $element ) {
 			 */
 			if ( $element->hasAttribute( Attribute::CHARSET ) ) {
 				$this->meta_tags[ self::TAG_CHARSET ][] = $element;
-			} elseif ( 'viewport' === $element->getAttribute( Attribute::NAME ) ) {
+			} elseif ( Attribute::VIEWPORT === $element->getAttribute( Attribute::NAME ) ) {
 				$this->meta_tags[ self::TAG_VIEWPORT ][] = $element;
-			} elseif ( 'amp-script-src' === $element->getAttribute( Attribute::NAME ) ) {
+			} elseif ( Attribute::AMP_SCRIPT_SRC === $element->getAttribute( Attribute::NAME ) ) {
 				$this->meta_tags[ self::TAG_AMP_SCRIPT_SRC ][] = $element;
+			} elseif (
+				Tag::NOSCRIPT === $element->tagName
+				&& $element->firstChild instanceof DOMElement
+				&& Tag::STYLE === $element->firstChild->tagName
+				&& $element->firstChild->hasAttribute( Attribute::AMP_BOILERPLATE )
+			) {
+				$this->meta_tags[ self::TAG_AMP_BOILERPLATE ][] = $element;
+			} elseif (
+				Tag::STYLE === $element->tagName
+				&& $element->hasAttribute( Attribute::AMP_BOILERPLATE )
+			) {
+				$this->meta_tags[ self::TAG_AMP_BOILERPLATE ][] = $element;
```

Comment:
It turns out this code will never match anything because `$elements` is only `meta` elements.